### PR TITLE
add gr8conf

### PR DESCRIPTION
**Pull request by sbglasius**
GR8Conf is an All Things Groovy conference running in Europe, US and India

### CODE REVIEW

**Comment by danfinlay** [comment]
`.ai` isn't a web format, could you re-submit as an `.svg`?

**Comment by sbglasius** [comment]
Updated to `.svg` vector

File: meta/gr8conf.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "gr8conf",
+  "author": "Soeren Glasius (GR8Conf Organizer)",
+  "license": "CC0",
+  "raster": "http://hexb.in/hexagons/gr8conf.png",
+  "vector": "http://hexb.in/vector/gr8conf.ai",
```
Comment (danfinlay) [inline meta/gr8conf.json]: Need to update this reference to point to the new `.svg` file.

**Review by danfinlay — CHANGES_REQUESTED** [review]

File: meta/gr8conf.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "description": "GR8Conf - The 'All Things Groovy' conference",
```
Comment (danfinlay) [inline meta/gr8conf.json]: Trailing comma is invalid JSON, will fail our build, too.  Sorry!

**Pull request merged**